examples/python/arkitscenes/main.py

```diff
@@ -0,0 +1,450 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import os
+from pathlib import Path
+from typing import Any, Dict, List, Tuple
+
+import cv2
+import matplotlib.pyplot as plt
+import numpy as np
+import numpy.typing as npt
+import rerun as rr
+import trimesh
+from download_dataset import AVAILABLE_RECORDINGS, ensure_recording_available
+from scipy.spatial.transform import Rotation as R
+from tqdm import tqdm
+
+# hack for now since dataset does not provide orientation information, only known after initial visual inspection
+ORIENTATION = {
+    "48458663": "landscape",
+    "42444949": "portrait",
+    "41069046": "portrait",
+    "41125722": "portrait",
+    "41125763": "portrait",
+    "42446167": "portrait",
+}
+assert len(ORIENTATION) == len(AVAILABLE_RECORDINGS)
+assert set(ORIENTATION.keys()) == set(AVAILABLE_RECORDINGS)
+
+
+def load_json(js_path: Path) -> Dict[str, Any]:
+    with open(js_path, "r") as f:
+        json_data = json.load(f)  # type: Dict[str, Any]
+    return json_data
+
+
+def log_annotated_bboxes(
+    annotation: Dict[str, Any]
+) -> Tuple[npt.NDArray[np.float64], List[str], List[Tuple[int, int, int, int]]]:
+    """
+    Logs annotated oriented bounding boxes to Rerun.
+
+    We currently calculate and return the 3D bounding boxes keypoints, labels, and colors for each object to log them in
+    each camera frame TODO(pablovela5620): Once #1581 is resolved this can be removed.
+
+    annotation json file
+    |  |-- label: object name of bounding box
+    |  |-- axesLengths[x, y, z]: size of the origin bounding-box before transforming
+    |  |-- centroid[]: the translation matrix (1,3) of bounding-box
+    |  |-- normalizedAxes[]: the rotation matrix (3,3) of bounding-box
+    """
+    bbox_list = []
+    bbox_labels = []
+    num_objects = len(annotation["data"])
+    # Generate an array of evenly spaced values from 0 to 1
+    color_positions = np.linspace(0, 1, num_objects)
+
+    # Create a colormap
+    colormap = plt.cm.get_cmap("viridis")
+
+    # Generate the array of colors (0-1 range) rgba
+    color_array_float = [colormap(pos) for pos in color_positions]
+
+    # Convert the color values from 0-1 to 0-255
+    color_list = [(int(r * 255), int(g * 255), int(b * 255), int(a * 255)) for r, g, b, a in color_array_float]
```

Comment:
```suggestion
    # Generate a color per object that can be reused across both 3D obb and their 2D projections
    # TODO(pablovela5620): Once #1581 or #1728 is resolved this can be removed
    color_positions = np.linspace(0, 1, num_objects)
    colormap = plt.cm.get_cmap("viridis")
    color_array_float = [colormap(pos) for pos in color_positions]
    color_list = [(int(r * 255), int(g * 255), int(b * 255), int(a * 255)) for r, g, b, a in color_array_float]
```